python/pyspark/sql/pandas/conversion.py

```diff
@@ -212,6 +212,9 @@ def toPandas(self):
                 if isinstance(field.dataType, TimestampType):
                     pdf[field.name] = \
                         _check_series_convert_timestamps_local_tz(pdf[field.name], timezone)
+                if isinstance(field.dataType, TimestampNTZType):
+                    pdf[field.name] = \
+                        _check_series_convert_timestamps_local_tz(pdf[field.name], timezone)
```

Comment:
Hm, I think we shouldn't localize it with timezone actually. Can we explicitly set `None` to the timezone?